zebra-state/src/service/finalized_state/disk_format/tests/prop.rs

```diff
@@ -4,26 +4,41 @@ use proptest::{arbitrary::any, prelude::*};
 
 use zebra_chain::{
     amount::NonNegative,
-    block::{self, Block},
+    block::{self, Block, Height},
     transparent,
     value_balance::ValueBalance,
 };
 
 use crate::service::finalized_state::{
     arbitrary::assert_value_properties,
-    disk_format::{transparent::OutputLocation, TransactionLocation},
+    disk_format::{block::MAX_ON_DISK_HEIGHT, transparent::OutputLocation, TransactionLocation},
 };
 
 #[test]
 fn roundtrip_block_height() {
     zebra_test::init();
-    proptest!(|(val in any::<block::Height>())| assert_value_properties(val));
+
+    proptest!(
+        |(mut val in any::<block::Height>())| {
```

Comment:
```suggestion
        |(mut val in any::<Height>())| {
```

Reply:
I think we can probably just let this one slide for now.

Reply:
Actually I fixed it in 20bf9ebe7, because I had to re-run CI anyway